forui/lib/src/widgets/tile/tile.dart

```diff
@@ -48,12 +48,23 @@ class FTile extends StatelessWidget with FTileMixin {
   /// Provide a style to prevent inheriting from the ancestor tile group's style.
   final FTileStyle? style;
 
-  /// Whether the FTile is enabled. Defaults to true.
-  final bool enabled;
+  /// Whether the `FTile` is enabled. Defaults to true.
+  final bool? enabled;
```

Comment:
It's because we propagate whether a tile is enabled down from the enclosing tile group. Defaulting it to `null` means "use enclosing tile group's value" while passing in a non-null value overrides the enclosing tile group's value.